docs/api/public_php_api.md

```diff
@@ -2,6 +2,10 @@
 
 The PHP API is also commonly referred to as the **public API**. Currently it exposes a Repository which allows you to create, read, update, manage and delete all objects available in eZ Platform. First and foremost content, but also related objects like Sections, Locations, Content Types, Content Type groups, languages and so on.
 
+!!! info "API"
+
+    An Application Programming Interface (API) allows you to connect your code to eZ Platform. [From the eZ Blog article](http://ez.no/Blog/How-would-you-explain-what-an-API-is-to-your-mom) you can learn basic idea behind it.
```

Comment:
I'd switch the order: You can learn the basic idea behind it from the eZ Blog article